reference/resource-definitions.md

```diff
@@ -441,4 +441,21 @@ to definitions found within the action block itself.
 NOTE: Currently `action_defaults` does not support sharing `routing` blocks. It 
 is probable, however, that this will be supported soon if the use case arises.
 
+## Canonical Paths
 
+You can specify which action should be used for the resource's canonical href with the `canonical_path` method:
+
+{% highlight ruby %}
+class Blogs
+  include Praxis::ResourceDefinition
+
+  canonical_path :show
+end
+{% endhighlight %}
+
```

Comment:
I'd add:

When none has been explicitly defined, the system will default the `canonical_path` of the resource to the `:show` action.